### PR TITLE
Add doc about opening k8s dashboard in Minikube tutorial

### PR DESCRIPTION
**Pull request by vhosakot**
Adding documentation about how to open the kubernetes dashboard using `minikube dashboard` in the Minikube tutorial https://kubernetes.io/docs/tutorials/stateless-application/hello-minikube/.

Fixes https://github.com/kubernetes/website/issues/2450.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7375)

### CODE REVIEW

File: docs/tutorials/stateless-application/hello-minikube.md [hunk 1]
```diff
@@ -116,6 +116,12 @@ Verify that `kubectl` is configured to communicate with your cluster:
 kubectl cluster-info
 ```
 
+Open the kubernetes dashboard GUI in browser:
+
```
Comment (steveperry-53) [inline docs/tutorials/stateless-application/hello-minikube.md]: Capitalize Kubernetes.
in browser -> in a browser
Suggestion: Remove GUI.

Open the Kubernetes dashboard in a browser:

Comment (vhosakot) [reply docs/tutorials/stateless-application/hello-minikube.md]: @steveperry-53 Thanks for the comment. I've addressed it. Please see latest commit.

**Pull request merged**